packages/client/components/RetroMeeting.tsx

```diff
@@ -76,7 +77,7 @@ const RetroMeeting = (props: Props) => {
   const localPhaseType = localPhase?.phaseType
   const isDemoStageComplete =
     meetingId === RetroDemo.MEETING_ID
-      ? ((atmosphere as unknown) as LocalAtmosphere).clientGraphQLServer.isBotFinished()
+      ? ((atmosphere as unknown) as LocalAtmosphere).clientGraphQLServer.isBotFinished() || !PROD
```

Comment:
this won't target testing specifically, perhaps window.Cypress instead? https://docs.cypress.io/faq/questions/using-cypress-faq.html#Is-there-any-way-to-detect-if-my-app-is-running-under-Cypress